source/img/security_comp_impl.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="app.diagrams.net" modified="2020-10-28T10:08:22.343Z" agent="5.0 (Macintosh; Intel Mac OS X 10_15_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.111 Safari/537.36" etag="46Q8G4Jed1K0Wz_FPPRc" version="13.8.5" type="device"><diagram id="PAKIylsEG8q9MCfg26HX" name="Page-1">3Vpbb+MqEP41kc55SOV7nMdc2t2VujqV+rBnn1bExpctNi7Gbby//oCN4wtO67Z2k5wqamAYCMzMNwwDM30T7b8QkATfsQvRTFPc/UzfzjRtaVjsPyfkJcHUjZLgk9AtSWpNuA//QEFUBDULXZi2GCnGiIZJm+jgOIYObdEAIfi5zeZh1P7VBPhQItw7AMnUH6FLA0G1FKVu+ApDPxA/bVcNEaiYBSENgIufGyT9eqZvCMa0LEX7DURcdpVcyn43R1oPEyMwpkM65Nk60EmOsl/X2/Wj/jWDN2huLMXkaF6tGLpMAKKKCQ2wj2OArmvqmuAsdiEfVmG1mucW44QRVUb8DSnNhTZBRjEjBTRColWeulhNijPiwJfma5eMfJKNnmLFXyCOICU5YyAQARo+tbUIhDH4B75D1zscsrloirBbaym0JsxWt5X2EBQQH1LRq5Y6KzSmUZMKXbxFL2KdTwBlYg2SotpqeA5CCu8TUIjvmWGxLXKAQj9mZQQ9JvK1C9Kg6PqiOp4goXD/opwrealteamWgPlzCzQlLWjgRdWs47ppSfWtIuyToIX44nes4PPCD7gLMH7gYoWELbZiYL934Dl0IhWFBoBbShj/Zg4nZSWcAD4Ec1QOIOnH9OSFCG0wwqToq7sA2p7D6Ckl+AE2WizHhjtP1izXWcj810qQKcfkGCo27A4kTEVWsdajYkuZSMPL1zV8h91hSkVgBxGb4GrGl9BtZd8g4nKMd2kpTgW7IMjmHsLPc+bgAkzCP8zh4LgcAsZgh6A7ZCSpXvRIExC3zMh6zHAxFHAe/MKk5k5pDfz3iL/7SzPNYvJMmEqn/DevFNP2cEznHohClB8WW06A2RTbmwvXvSMgjLllRzjGEsNhYN6aFpbcGj0t3D4fe3llJ7SeeiUDhpd5glHo5GwJsRf68wgk8xhEohtIEtHe0F0pkEqIH4FYP0I6MBoBMIdQoPKJhgwY3egBjK5OhZhq4AZkVknCCBumOKZz7gLH2GUcJjU21ihiXHa2Yk0/td/RTxI0MQmS/N9m5Scf7Moyq/p2L0Yva3mzdgdJyFbPlbJVPhSCqVUYXkRALzFqk8RqK0JA3mBIeAyWHg/lVEXvbFwi6L15ZwdWKOcwbsRnXFYkPtwMlLHN4GMusHJAtQv8lrKNm4uB4H1+/g7w9IFXtWldjK1qQ21VPS9b1SRb/ac4a9yXZ42RbXXsQ6Hd9aMnN1zVOunOrVyZrb37kzdufSgKjPNCgS6hYOVQLFt/GoCEF7MIlQx1rH/Lj3d3OA3LI9p2hynF0dHDQEN5OKOIBcabQ35RmcjNa904t+e08KlYsS/MyRtDzXtxXuZtSOYtCb4hoIOTPnj4tO3hOxkkzV6Y11pfBskr/liLiKAZ0Vyzj3KlKCY74bOjhc6/ZuaWn/d5g1qQtQ512UsthuhyLo8MvCh6s/aeQdQOTbM7vOzDlifAH+19fhFx9ZDtIImZ40yvQodjfp2Qb0VhC5JwKgx3cmTqQgax2QPiKt07unFpiwsD8WIgiEe/B/gYhhc9GD6WbB6Sjiz2qzIbNjDReDQSVN98OdC/J749Mz0CoNQBZ5/P3RZ1/bQhZDv581ru5/1I1Iee7/XR0zwf0498vpegeB0/4fwmRHTolQ8HXtxP/vMC9C76vsfqpK97stefnHY9SYbsWNp1QuSdLlvxvgTrIcoRlrLoPER4jb+6XJ80v1pJtZv3e2nrPFHCz+aXd2eFvAuLInXrwhCkmB1fq78CoVc6TAQhS4LQ/zERoyp6G39mT9D5qTGnIWcEzhp/9lD4nVfoaJiSfbswQTiXI7+kfldSE0P+YiMhmDL75EcyhT9pEA8kuupDKExS2MhbOAhn7okumDpJC6XnjNW33ywms3fZ0Ugx/KZ4OfKdyafvjY0U09evS8TLExd6IEN0nN1fxOJvD+ani961jlLtUwcRxoAcybuV2nwyNJ1CPc/TnF6FutbOMq1JFWq0Fdp3PzCZQnfQJfPb29W9T9eW/lNdrR1vLgfUIOYTDihNCp/9mMFUBti7Xd8Ynm7Riax78rO9z1TNtwuRVevH3WUIVr+Q16//Aw==</diagram></mxfile>
```

Comment:
Is it required?

Reply:
It allows you to edit the image in draw.io editor